charts/opentelemetry-kube-stack/templates/_config.tpl

```diff
@@ -0,0 +1,443 @@
+{{- $_ := set $config.service.pipelines.metrics "receivers" (append ($config.service.pipelines.metrics.receivers | default list) "k8s_cluster" | uniq)  }}
+{{- end }}
+{{- $config | toYaml }}
+{{- end }}
+
+{{- define "opentelemetry-kube-stack.collector.clusterMetricsConfig" -}}
+receivers:
+  k8s_cluster:
+    collection_interval: 10s
+    auth_type: serviceAccount
+    node_conditions_to_report: [Ready, MemoryPressure, DiskPressure, NetworkUnavailable]
+    allocatable_types_to_report: [cpu, memory, storage]
+{{- end }}
+
+{{- define "opentelemetry-kube-stack.collector.applyKubeletMetricsConfig" -}}
+{{- $config := mustMergeOverwrite (include "opentelemetry-kube-stack.collector.kubeletMetricsConfig" .collector | fromYaml) .collector.config }}
+{{- if and (dig "service" "pipelines" "metrics" false $config) (not (has "kubeletstats" (dig "service" "pipelines" "metrics" "receivers" list $config))) }}
+{{- $_ := set $config.service.pipelines.metrics "receivers" (append ($config.service.pipelines.metrics.receivers | default list) "kubeletstats" | uniq)  }}
+{{- end }}
+{{- $config | toYaml }}
+{{- end }}
+
+{{- define "opentelemetry-kube-stack.collector.kubeletMetricsConfig" -}}
+receivers:
+  kubeletstats:
+    collection_interval: "15s"
+    auth_type: "serviceAccount"
+    insecure_skip_verify: true
+    # For this scrape to work, the RBAC must have `nodes/stats` GET access.
+    endpoint: "https://${env:OTEL_K8S_NODE_IP}:10250"
+    extra_metadata_labels:
+    - container.id
+    - k8s.volume.type
+    metric_groups:
+    - node
+    - pod
+    - volume
+    - container
+    k8s_api_config:
+        auth_type: serviceAccount
+    metrics:
+        # k8s.pod.cpu.utilization is being deprecated
+        k8s.pod.cpu.usage:
+            enabled: true
+        container.cpu.usage:
+            enabled: true
+        k8s.node.cpu.usage:
+            enabled: true
+        k8s.node.uptime:
+            enabled: true
+        k8s.pod.uptime:
+            enabled: true
+{{- end }}
+
+{{- define "opentelemetry-kube-stack.collector.applyLogsCollectionConfig" -}}
+{{- $config := mustMergeOverwrite (include "opentelemetry-kube-stack.collector.logsCollectionConfig" .collector | fromYaml) .collector.config }}
+{{- if and (dig "service" "pipelines" "logs" false $config) (not (has "filelog" (dig "service" "pipelines" "logs" "receivers" list $config))) }}
+{{- $_ := set $config.service.pipelines.logs "receivers" (append ($config.service.pipelines.logs.receivers | default list) "filelog" | uniq)  }}
+{{- end }}
+{{- if .collector.presets.logsCollection.storeCheckpoints}}
+{{- $_ := set $config.service "extensions" (append ($config.service.extensions | default list) "file_storage" | uniq)  }}
+{{- end }}
+{{- $config | toYaml }}
+{{- end }}
+
+{{- define "opentelemetry-kube-stack.collector.logsCollectionConfig" -}}
+{{- if .presets.logsCollection.storeCheckpoints }}
+extensions:
+  file_storage:
+    directory: /var/lib/otelcol
+{{- end }}
+receivers:
+  filelog:
```

Comment:
this can use the container-logs parser now, the collector chart has been updated